lib/oDataClient.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const RestClient = require('./restClient');
+
+module.exports = function (emitter, cfg) {
```

Comment:
maybe it's better to use ES2015 class syntax here since you're instantiating oDataClient with `new`?